src/main/kotlin/uk/gov/dluhc/printapi/exception/GlobalExceptionHandler.kt

```diff
@@ -135,17 +135,17 @@ class GlobalExceptionHandler(
     override fun handleMethodArgumentNotValid(
         e: MethodArgumentNotValidException,
         headers: HttpHeaders,
-        status: HttpStatus,
+        status: HttpStatusCode,
         request: WebRequest,
-    ): ResponseEntity<Any> {
+    ): ResponseEntity<Any>? {
         return populateErrorResponseAndHandleExceptionInternal(e, BAD_REQUEST, request)
     }
 
     private fun populateErrorResponseAndHandleExceptionInternal(
         exception: Exception,
-        status: HttpStatus,
+        status: HttpStatusCode,
         request: WebRequest,
-    ): ResponseEntity<Any> {
+    ): ResponseEntity<Any>? {
         request.setAttribute(ERROR_STATUS_CODE, status.value(), SCOPE_REQUEST)
         val body = errorAttributes.getErrorResponse(request)
         return handleExceptionInternal(exception, body, HttpHeaders(), status, request)
```

Comment:
Worth figuring out why these are nullable now

Reply:
Yep does need to be nullable - `handleExceptionInternal` comes from a spring package and has a @Nullable annotation that I don't think was respected in SB2, but now gives a type error if we don't make the response nullable